src/Umbraco.Web/Trees/ApplicationTreeController.cs

```diff
@@ -71,9 +86,64 @@ public async Task<SectionRootNode> GetApplicationTrees(string application, strin
                 }
             }
 
-            var multiTree = SectionRootNode.CreateMultiTreeSectionRoot(rootId, collection);
-            multiTree.Name = Services.TextService.Localize("sections/"+ application);
-            return multiTree;
+            //Don't apply fancy grouping logic futher down, if we are not 'settings' section
+            if(application != Constants.Applications.Settings)
+            {
+                var multiTree = SectionRootNode.CreateMultiTreeSectionRoot(rootId, collection);
+                multiTree.Name = Services.TextService.Localize("sections/" + application);
+
+                rootNodeGroups.Add(multiTree);
+                return rootNodeGroups;
+            }
+
+            //For settings section only
+            //Group trees by [CoreTree] attribute
+
+            //Core Trees contains all trees for all sections/applications
+            foreach(var treeSectionGroup in CoreTrees.Value)
+            {
+                var treeGroupName = treeSectionGroup.Key;
+
+                var groupNodeCollection = new TreeNodeCollection();
+
+                //Only add trees to a new collection if they are from 'settings'
+                foreach (var treeItem in treeSectionGroup)
+                {
+                    //Item1 tuple - is the type from all tree types
+                    var treeItemType = treeItem.Item1;
+
+                    var findAppTree = appTrees.SingleOrDefault(x => x.GetRuntimeType() == treeItemType);
```

Comment:
FirstOrDefault is faster than SingleOrDefault since SingleOrDefault must scan the entire collection, FirstOrDefault will exit the loop once one is found